lib/all_changed_attributes.rb

```diff
@@ -0,0 +1,47 @@
+require 'all_changed_attributes/version'
+
+module AllChangedAttributes
```

Comment:
All constants should be nested under the gem's top-level constant ArTransactionChanges to avoid polluting the global namespace.